Travel Scheduler/Views/PlaceView.m

```diff
@@ -102,9 +103,7 @@ - (instancetype)initWithFrame:(CGRect)frame andPlace:(Place *)place
     _place = place;
     [self makeLabels];
     [self makeEditButton];
-    self.lockImage = instantiateLockImageView(self.timeRange);
```

Comment:
lol I also did that. We might run into merge conflicts, but it should be easy to fix